code/lib/cli/src/generators/NUXT/index.ts

```diff
@@ -0,0 +1,32 @@
+import { baseGenerator } from '../baseGenerator';
+import type { Generator } from '../types';
+
+const generator: Generator = async (packageManager, npmOptions, options) => {
+  await baseGenerator(
+    packageManager,
+    npmOptions,
+    options,
+    'nuxt',
+    {
+      extraPackages: async ({ builder }) => {
+        return ['@nuxtjs/storybook'];
+      },
+      installStorybookPackage: false,
```

Comment:
Can you explain why we need those options only fo the nuxt package?

Reply:
The setup with nuxt is a bit different. In the nuxt world, you normally install a so-called module. The task of the module is to interface with whatever external library is used under the hood, as well as shipping the library (i.e. it is declared as a dependency). 

For example, the tailwind module for nuxt (https://github.com/nuxt-modules/tailwindcss) declares `tailwindcss` as a dependency (https://github.com/nuxt-modules/tailwindcss/blob/main/package.json#L56). Similarly, in the storybook module we already pull-in storybook as the dependency: https://github.com/nuxt-modules/storybook/blob/30cb7036f1bfc5bc50b11b6b7c3857bcf23886b6/package.json#L81 Thus users don't need to add the storybook packages themselves in their `package.json`.